File: w01/d02/Chris_Heuberger/calculator.rb
# A user should be given a menu of operations
# A user should be able to choose from the menu
def menu
  puts "Which calculator would you like to use?"
  puts "Choose one: basic or advanced?"
  calc_choice = gets.chomp
  if calc_choice == "basic"
    basic–calc
  elsif calc–choice == "advanced"
    advanced–calc
  end
end

def basic_calc
  print "(a)dd, (s)ubtract, (m)ultiply, (d)ivide: "
  operation = gets.chomp
  puts "What's your first number?"
  n1 = gets.chomp!.to_i
  puts "What's your second number?"
  n2 = gets.chomp!.to_i

  case
    when operation == "a"
      puts "You chose addition."
      add_answer = first_number + second_number
      puts add_answer
    when operation == "s"
      puts "You chose subtraction."
      sub_answer = first_number - second_number
      puts sub_answer
    when operation == "m"
      puts "You chose multiplication"
      multip_answer = first_number * second_number
      puts multip_answer
    when operation == "d"
      puts "You chose division."
      div_answer = first_number / second_number
      puts div_answer
    else
      puts "That's not a choice ding dong! Try again."
    end

end

end

def advanced_calc
  print "(p)ower, (s)qrt: "

end

response = menu

# This process should continue until the user selects a quit option from the menu
while response != 'q'
  response = menu
end

File: w06/d01/Wake_Lankard/calculator.js
function calculator(x, y, func){
  var output;

  switch(func) {
    case 'addition':
      output = x + y;
      break;
    case 'subtraction':
      output = x - y;
      break;
    case 'multiplication':
      output = x * y;
      break;
    case 'division':
      if (y != 0){
        output = x / y;
      } else {
        output = "division by zero error.";
      }
      break;
    default:
      output = "invalid input.";
  }

  return output;
}

console.log("3 + 2 = " + calculator(3,2,"addition"));
console.log("3 - 2 = " + calculator(3,2,"subtraction"));
console.log("3 * 2 = " + calculator(3,2,"multiplication"));
console.log("3 / 0 = " + calculator(3,0,"division"));
console.log("6 - 2 = " + calculator(6,2,"division"));

